kafka_topic-transferer/kafka-topic-transferer.go

```diff
@@ -0,0 +1,79 @@
+package transferer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+	"time"
+
+	kf "github.com/codeuniversity/smag-mvp/kafka"
+	"github.com/codeuniversity/smag-mvp/models"
+	"github.com/codeuniversity/smag-mvp/utils"
+	"github.com/codeuniversity/smag-mvp/worker"
+
+	"github.com/segmentio/kafka-go"
+)
+
+// Transferer represents the Transferer containing all clients it uses
+type Transferer struct {
+	fromTopic *kafka.Reader
+	toTopic   *kafka.Writer
+
+	*worker.Worker
```

Comment:
FYI: Let's have the included fields first in the struct definition (since they are "similar" to extending a class in other languages)